.github/workflows/test.yml

```diff
@@ -101,3 +109,35 @@ jobs:
     - name: Build crm
       run: make build-crm
 
+  greenframe:
+    runs-on: ubuntu-latest
+    name: GreenFrame
+    needs: [e-commerce]
+    if: success() && github.ref == 'refs/heads/master'
+    steps:
+      # To use this repository's private action,
+      # you must check out the repository
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch all the git history to enable git comparison
```

Comment:
I'm not 100% sure, but it seems that this is only required to compare the result of the current branch with the base branch. But since we're only willing to run the analysis on `master` (and not on feature branches), I have limited the depth to 2 to fetch only the current commit and the one before. Hence I'm not sure we can benefit from the 'diff' feature, but at least the checkout will be faster.